igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -157,9 +160,11 @@ class JenkinsBuildMonitor extends CommonPollingMonitor<JobDelta, JobPollingDelta
 
         } catch (e) {
             log.error("Error processing builds for [{}:{}]", kv("master", master), kv("job", job.name), e)
-            if (e.cause instanceof RetrofitError) {
-                def re = (RetrofitError) e.cause
-                log.error("Error communicating with jenkins for [{}:{}]: {}", kv("master", master), kv("job", job.name), kv("url", re.url), re)
+            if (e instanceof SpinnakerServerException) {
+                def re =  e.getRetrofitError();
```

Comment:
I'd really like to leave any use of retrofit1 classes behind here, so we can upgrade to retrofit2, remove retrofit1, and remove exposure to some CVEs.  At the moment, [SpinnakerServerException](https://github.com/spinnaker/kork/blob/8433e316bdcc7611c8bd64e5611c84ca7fedebb7/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerServerException.java) doesn't have a url attribute, but it could, and then `e.url` would work here.